README.md

```diff
@@ -38,20 +38,27 @@ PRs should be posted _before_ class begins on Tuesday, January 2, 2024.
 ### Interview Practice Questions
 
 1. What are three (3) syntax difference between HTML and JSX?
-
+   //class vs className
+   //inline styles
+   //self-closing tags
 2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
-
+  //It is a popular package manager for Node.js applications.
+   //When I run it, it installs package.json file so that the source can be used and shared.
+   //It also updates yarn.lock file.
 3. What is the difference between state values and props in React?
-
+   //State is used when we set a initial value for the variable with using useState.
+   //It is changable.
+   //Props is used when we want to use the function over and over again with same information by creating another JS file under SCR.
```

Comment:
You are right about state! But props are a communication pipeline. Functions can always be used over an over. But you can pass a function as props to you can call in another component.